src/full/Agda/Utils/Tuple.hs

```diff
@@ -1,33 +1,46 @@
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 
-module Agda.Utils.Tuple where
+module Agda.Utils.Tuple
+  ( (-*-)
```

Comment:
Ah, a heroic effort to put export lists to the utility modules. :D
In fact, for utility modules that export all of their functions we usually do not maintain export lists, as it is more of a hassle than a productivity gain.  
I typically add a function to a utility module but forget to add it to the export list.  20 min later the compiler gives me a scope error.  I have to go back and add the function to the export list and wait another 20 min for the compiler...
So, don't invest too much time in writing export lists.  Someone might chop it away again at some later time in frustration.

Reply:
Thanks for the comment. I want to re-export functions `isLeft` and `isRight` from `Data.Either`. I am not aware of any other way to do this.